scoap3/modules/records/util.py

```diff
@@ -168,14 +168,16 @@ def create_from_json(records, apply_async=True):
         queue = current_app.config['CRAWLER_CELERY_QUEUE']
 
         if apply_async:
-            start.apply_async(
+            workflow_id = start.apply_async(
```

Comment:
So, currently this change does not really change anything from our infrastructure perspective, because celery is not really running more than one task...but doing this, you are creating an async task and waiting for it's result, which means you are kind of forcing this code to be sync, because the current process is waiting for celery to return the result of the task itself. Is this what you want?

Reply:
Hm, sadly you're right and this wan't my plan and we definitely have to change this. I wanted to make the async and sync versions behave the same.
For the sync version, it's really useful (mostly for the tests) to have the created workflow id. Without that I can't check what happened with the workflow. For the async part, the return value is not used. In this case do you think I should just return the object returned by the apply_async function, or something else?
Also, this function is not used in production, only when we want to upload something manually and in the tests.

Reply:
In this case I would say it's ok to let it return :+1: